### ISSUE TITLE
TextBox error messages don't disappear when going to the next slide in transitioner

### ISSUE THREAD

**Issue by Quadrition** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1334]
I have multiple Controls Binded in the Transitioner and every control have textboxes in it. When I have for example 2 slides (controls), and on the first slide I enter values in the TextBoxes so I don't have error messages anymore, then I switch to the second slide, where there are textboxes with error messages, and then switch back to the first slide, the error messages stays on the first slide, although the values in the first slide textboxes are valid. Here is the video for the simpler explanation.

[transitioner_bug.zip](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/files/3436049/transitioner_bug.zip)

Here is repository to the project, but you must set up a database before starting it, so it is just for the code watching: https://github.com/Quadrition/Instrument-Demo

**Comment by Keboo** [comment]
Duplicate of #695 
The reason for this behavior is that validation errors are shown in the [adorner layer](https://docs.microsoft.com/en-us/dotnet/framework/wpf/controls/adorners-overview), which sits on top of the normal WPF controls.
